webapp/channels/src/components/user_settings/modal/user_settings_modal.tsx

```diff
@@ -343,8 +343,8 @@ class UserSettingsModal extends React.PureComponent<Props, State> {
                 onHide={this.handleHide}
                 onExited={this.handleHidden}
                 enforceFocus={this.state.enforceFocus}
-                role='dialog'
                 aria-label={modalTitle}
+                role='none'
```

Comment:
Is role required? Or can we just omit it? Or omitting it would default to dialog?

Reply:
omitting will default to dialog.